<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>毕业选题管理系统-登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>501</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color : rgba(88, 108, 152, 240);
	border-radius : 12px;

}




QLineEdit
{
	border-radius : 6px;
	font-size : 16px;
}

QRadioButton::indicator:checked { background-color: rgb(241, 196, 15); }

QRadioButton::indicator:hover { background-color: qradialgradient(cx:0.5, cy:0.5, radius: 1.5, fx:0.5, fy:0.5, stop: 0 rgb(241, 196, 15), stop: 1 rgb(213, 175, 11)); }

QRadioButton::indicator { border-radius: 50%; border: 2px solid rgb(44, 62, 80); }

QRadioButton::indicator:pressed, QRadioButton::indicator:checked:pressed { background-color: rgb(213, 175, 11); }

QRadioButton { color: rgb(44, 62, 80); font-size: 18px; }

QPushButton { background-color: rgb(241, 196, 15); color: rgb(44, 62, 80); font-size: 16px; border-radius: 12px; border: none; padding: 8px 16px; }



QPushButton:hover { background-color: rgb(213, 175, 11); }

QLabel
{
background-color : rgba(88, 108, 152, 0);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="pwdLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="stuRadioButton">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学生</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="teaRadioButton">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>教师</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="admRadioButton">
        <property name="text">
         <string>管理员</string>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>271</width>
       <height>130</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="3" column="1">
       <widget class="QPushButton" name="loginBtn">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="exitBtn">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="accountLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
    <widget class="QLabel" name="accountLable">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>72</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="pwdLable">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
